corporate/lib/stripe.py

```diff
@@ -966,9 +966,11 @@ def update_sponsorship_status(
         acting_user=acting_user,
         event_type=RealmAuditLog.REALM_SPONSORSHIP_PENDING_STATUS_CHANGED,
         event_time=timezone_now(),
-        extra_data={
-            "sponsorship_pending": sponsorship_pending,
-        },
+        extra_data=str(
+            {
+                "sponsorship_pending": sponsorship_pending,
+            }
+        ),
```

Comment:
If you delete the [magic trailing comma](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma) after `sponsorship_pending`, Black will format this more concisely:

```python
        extra_data=str({"sponsorship_pending": sponsorship_pending}),
```